### PR TITLE
ADEN-5660 differ AN placement IDs when recovered traffic

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/ADEN-5660

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js [hunk 1]
```diff
@@ -9,19 +9,13 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
 	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements',
 		placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig;
 
-	function getPlacement(skin, position) {
-		var vertical = adContext.getContext().targeting.mappedVerticalName;
+	function getPlacement(skin, position, recovery) {
+		var vertical = recovery ? 'recovery' : adContext.getContext().targeting.mappedVerticalName;
 
 		if (placementsMap && placementsMap[skin] && placementsMap[skin][vertical]) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js]: MAJOR Merge this if statement with the nested one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/spec/lookup/prebid/adapters/appnexusPlacements.spec.js [hunk 2]
```diff
@@ -90,14 +93,14 @@ describe('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', functio
...
+			mocks.log
 		);
 	}
 
-
 	testCases.forEach(function (testCase) {
-		it('expected placementId is returned for skin/vertical combination ', function () {
-			var appNexusPlacements = getModule(testCase.vertical, testCase.env),
+		it('expected placementId is returned for skin/vertical combination ' + testCase.vertical + ':' + testCase.env, function () {
+			var appNexusPlacements = getModule(testCase.vertical, testCase.env, false),
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/spec/lookup/prebid/adapters/appnexusPlacements.spec.js]: CRITICAL "getModule" expects 2 arguments, but 3 were provided. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS930)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js [hunk 3]
```diff
@@ -99,8 +99,8 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexus',[
 		return geo.isProperGeo(instantGlobals.wgAdDriverAppNexusBidderCountries);
 	}
 
-	function prepareAdUnit(slotName, config, skin) {
-		var placementId = appnexusPlacements.getPlacement(skin, config.position);
+	function prepareAdUnit(slotName, config, skin, recovery) {
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexus.js]: What do you think about more descriptive variable name? `isRecoveryEnabled`?

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**